language/move-vm/types/src/values/values_impl.rs

```diff
@@ -1219,6 +1361,16 @@ impl VMValueCast<u128> for IntegerValue {
     }
 }
 
+impl VMValueCast<move_core_types::u256::U256> for IntegerValue {
```

Comment:
Yeah should probably just import `move_core_types::u256::U256` at the top to avoid repeating it many times, but this is a minor problem we can fix later.

Reply:
Yeah there's already `U256` used for `IntegerValue::U256`, so I wanted to disambiguate.
But I will use `u256::U256` instead.